native/cocos/bindings/jswrapper/jsvm/Object.cpp

```diff
@@ -138,13 +141,19 @@ bool Object::isArray() const {
     JSVM_Status status;
     bool ret = false;
     NODE_API_CALL(status, _env, OH_JSVM_IsArray(_env, _objRef.getValue(_env), &ret));
+    if (status != JSVM_OK) {
```

Comment:
Can we use JSVM_CALL_BASE?
![image](https://github.com/user-attachments/assets/8b3bd360-02ae-4937-900f-de79e5881db6)